packages/wallet-sdk/jest.setup.ts

```diff
@@ -9,17 +9,3 @@ global.TextEncoder = TextEncoder;
 
 // @ts-expect-error Use util TextDecoder
 global.TextDecoder = TextDecoder;
-
-expect.extend({
-  toThrowEIPError(received, code, message = expect.any(String)) {
-    const expected = expect.objectContaining({
-      code,
-      message,
-    });
-    return {
-      pass: this.equals(received, expected),
-      message: () =>
-        this.utils.printDiffOrStringify(expected, received, 'Expected', 'Received', true),
-    };
-  },
-});
```

Comment:
this custom helper started causing some tsc typecheck bugs with newer yarn version for some reason.
deprecating and replacing with standard `toMatchObject`